webpay/pay/tests/test_tasks.py

```diff
@@ -392,7 +392,7 @@ def setUp(self):
                             'icons': {'64': 'http://app/i.png'},
                             'name': 'Virtual Sword',
                             'description': 'A fancy sword'}}}
-        self.prices = {'prices': [{'price': 1, 'currency': 'EUR'}]}
+        self.prices = {'prices': [{'price': '0.89', 'currency': 'EUR'}]}
```

Comment:
here you are sending it as str and over here: https://github.com/mozilla/solitude/pull/317/files#diff-c699da5bcbd06b92e39a87e0568a2df5R242 doing a multiplication... we aren't going to multiply a string I take it?